pisek/task_jobs/generator/generator_manager.py

```diff
@@ -115,17 +115,36 @@ class TestcaseInfoMixin(JobManager):
     def __init__(self, name: str, **kwargs) -> None:
         self.inputs: dict[str, tuple[set[int], int | None]] = {}
         self.input_dataset: set[InputPath] = set()
-        self._gen_inputs_job: dict[Optional[int], GenerateInput] = {}
+        self._last_testcase_job: dict[Optional[int], Job] = {}
```

Comment:
What does "last testcase job" mean? The last job to run for this solution and testcase, so likely judging?